Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -718,20 +735,31 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
     /// cases on the generated Swift enums.
     /// Defaultss to ``ApolloCodegenConfiguration/CaseConversionStrategy/camelCase``
     public let enumCases: CaseConversionStrategy
+    
+    /// Determines how the names of fields in the GraphQL schema will be converted into
+    /// properties in the generated Swift code.
+    /// Defaultss to ``ApolloCodegenConfiguration/CaseConversionStrategy/camelCase``
+    public let fieldCasing: CaseConversionStrategy
 
     /// Default property values
     public struct Default {
       public static let enumCases: CaseConversionStrategy = .camelCase
+      public static let fieldCasing: CaseConversionStrategy = .default
```

Comment:
Can we rename this to `fieldAccessors`. `enumCases` is referring to the cases of an enum, not to upper"case"/lower"case". This is the `CaseConversionStrategy` for `fieldAccessors`.